### PR TITLE
https://issues.redhat.com/browse/ACM-4062 dr4hub arch

### PR DESCRIPTION
**Pull request by oafischer**
Need to fix dr4hub links in known issues still.

### CODE REVIEW

File: backup_restore/backup_activation.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
+[#managed-cluster-activation-data]
+= Managed cluster activation data
+
```
Comment (swopebe) [inline backup_restore/backup_activation.adoc:3]: If this is a task, and it seems it is, we may want to indicate that with a gerund as we are trying to do throughout the content.

File: backup_restore/backup_hub_config.adoc [hunk 2]
```diff
@@ -0,0 +1,86 @@
+[#dr4hub-config]
+= Hub cluster configuration
```
Comment (swopebe) [inline backup_restore/backup_hub_config.adoc]: This title could be a title for any feature within RHACM or OCP so I would try a more specific approach here:
Configuring active or passive hub cluster (remember the gerund)

File: backup_restore/backup_install.adoc [hunk 3]
```diff
@@ -0,0 +1,176 @@
+[#install-backup-and-restore]
+= Installing the backup and restore operator
```
Comment (swopebe) [inline backup_restore/backup_install.adoc:2]: like this ^^^

File: backup_restore/backup_msa.adoc [hunk 4]
```diff
@@ -0,0 +1,120 @@
+[#auto-connect-clusters-msa]
+= Automatically connecting clusters by using a Managed Service Account
+
```
Comment (swopebe) [inline backup_restore/backup_msa.adoc:3]: Connecting clusters automatically

File: backup_restore/backup_schedule.adoc [hunk 5]
```diff
@@ -0,0 +1,222 @@
+[#using-backup-restore]
+= Scheduling and restoring backups
+
```
Comment (swopebe) [inline backup_restore/backup_schedule.adoc:3]: see these are gerunds and for the most part on titles, sections, we want to try to be parallel unless we are dealing with a concept or reference.

**Review by swopebe — CHANGES_REQUESTED** [review]
Couple changes to titles--I didn't review content. 

I see you have an intro but didn't see it here....that will link to all this I assume?

Thanks!

**Comment by oafischer** [comment]
Implemented the suggestions, thanks for the quick review. Yes, the intro topic has all the new structure with links. I think you need to click "load diff" to see it.

**Review by swopebe — APPROVED** [review]

**Review by birsanv — APPROVED** [review]
lgtm

**Pull request merged**